rl4co/models/nn/utils.py

```diff
@@ -55,17 +58,50 @@ def random_policy(td):
     return td
 
 
-def rollout(env, td, policy):
+def rollout(env, td, policy, max_steps: int = 10_000):
     """Helper function to rollout a policy. Currently, TorchRL does not allow to step
     over envs when done with `env.rollout()`. We need this because for environements that complete at different steps.
     """
     actions = []
+    steps = 0
     while not td["done"].all():
         td = policy(td)
         actions.append(td["action"])
         td = env.step(td)["next"]
+        steps += 1
+        if steps > max_steps:
+            break
     return (
         env.get_reward(td, torch.stack(actions, dim=1)),
         td,
         torch.stack(actions, dim=1),
     )
+
+
+class RandomPolicy(nn.Module):
+
+    """
+    Random Policy Class that randomly select actions from the action space
+    This policy can be useful to check the sanity of the environment during development
+
+    We match the function signature of forward to the one of the AutoregressivePolicy class
+
+    """
+
+    def __init__(self, env_name=None):
+        super().__init__()
+        self.env_name = env_name
+
+    def forward(
+        self,
+        td: TensorDict,
+        env: Union[str, RL4COEnvBase] = None,
```

Comment:
How about setting the`max_steps` to `inf` by default?
Since there may be cases in which one wants to test a very large scale problem, but this `max_steps` is kinda hardcoded. Or, perhaps we can log a warning if this happens

Reply:
I set the default value of `max_steps` as `None`. If `max_steps=None`, rollout decodes until it finds the environments are done.